### ISSUE TITLE
Duplicate request

### ISSUE THREAD

**Issue by syarra** [laurentheirendt/devTools#30]
After a call to `contribute` with option `3`:
```
      Available features are:
      -   allo

   -> Please enter the name of the feature that you want to submit/publish (example: add-constraints): allo
dev>   -> Your <allo> feature (branch) exists locally, but not remotely. Do you want to have your local <allo> published? Y/N [Y]:Y
dev>   -> Your <allo> feature (branch) exists locally, but not remotely. Do you want to have your local <allo> published? Y/N [Y]:
```
Why the last line is displayed twice?

**Comment by laurentheirendt** [comment]
I cannot reproduce this.
```

   -> Please select what you want to do (enter the number): 3
dev>  Your Github username is: laurentheirendt.  (Success) 
dev>  Your Github email is: laurent.heirendt@uni.lu.  (Success) 
dev>   -- Configuration -------- 
dev>      GitHub username:       laurentheirendt
dev>      Local directory :      [...]
dev>      Remote fork URL:       https://github.com/laurentheirendt/cobratoolbox.git
dev>      Remote repository URL: https://github.com/opencobra/cobratoolbox.git

      Available features are:
      -   allo

   -> Please enter the name of the feature that you want to submit/publish (example: add-constraints): allo
dev>   -> Your <allo> feature (branch) exists locally, but not remotely. Do you want to have your local <allo> published? Y/N [Y]:y
dev>   -> Do you want to open a pull request (PR) and submit your feature (branch) <allo>? Y/N [N]: n
dev>  You can open a pull request (PR) later using "openPR('allo')".
```

**Comment by laurentheirendt** [comment]
Solved by PR #35 . Thanks @syarra 🎉
